hack/test.sh

```diff
@@ -0,0 +1,15 @@
+E2E_K8S_VERSION="v1.22.2"
+IP_FAMILY="ipv4"
+cat <<EOF | kind create cluster \
+    --name lol           \
```

Comment:
lets keep a better naming :) if we have more cluster better to not call it  `lol2` or `rofl`